ui/src/components/Document/DocumentItemCover.js

```diff
@@ -6,8 +6,12 @@ import { Item } from 'semantic-ui-react';
 import isEmpty from 'lodash/isEmpty';
 
 export default class DocumentItemCover extends Component {
-  getRestrictions = meta => {
-    if (!isEmpty(meta) && meta.open_access) return null;
+  getRestrictions = document => {
+    if (
```

Comment:
I am trying to remove this behaviour of accepting both `metadata` and `document.metadata`, I think we should fix it when used.
So you could you please:
1. define the propTypes
```
DocumentItemCover.propTypes = {
  metadata: PropTypes.object.isRequired,
};
```
2. fix any usage of `DocumentItemCover` and make sure we always pass the `metadata`, and never the `document`
3. change this code to just do:
```
getLabel = metadata => {
  const isRestricted = _get(metadata, 'open_access', true);
  return isRestricted ? { ... } : null;
}
```